src/mvba/hash.rs

```diff
@@ -16,12 +16,12 @@ pub struct InvalidLength {
 }
 
 impl Hash32 {
-    pub fn calculate(data: &[u8]) -> Self {
+    pub fn calculate(data: impl AsRef<[u8]>) -> Self {
```

Comment:
I didn't know we can do this. It parse string as bytes vector. interesting

Reply:
yeah, we can pass any type to this function that implements `AsRef<[u8]>` trait.